cmds/task/actors.go

```diff
@@ -149,6 +210,38 @@ func runComplete(credentials *tcclient.Credentials, args []string, out io.Writer
 		return fmt.Errorf("could not complete the task %s: %v", taskID, err)
 	}
 
+	
+	if confirmrunComplete {
+		var response = confirmMsg("Completes", credentials, args, out, flagSet)
+		if response == "y" {
```

Comment:
If this is the same in each call to `confirmMsg`, the check for `y`, `Y`, `Yes` and other variants could be in the `confirmMsg` function

Reply:
The variable is a different one in each case.

Reply:
it's always "y'..